common/core/overlaybuffer.cpp

```diff
@@ -26,6 +26,11 @@
 
 #include "hwctrace.h"
 
+#ifdef USE_MINIGBM
+// minigbm specific DRM_FORMAT_YVU420_ANDROID enum
```

Comment:
Ha didnt realise we are defining it here, in this case we can remove this ifdef ?. With that removed you can merge the patch Tapani